### DISCUSSION TITLE
Allow maintainer to mark a comment as an answer

### DISCUSSION THREAD

**Discussion by olets** [community/community#70108] [Discussions] [answered]
### Select Topic Area

Bug

### Body

According to https://github.com/orgs/community/discussions/3190#discussioncomment-294747

> the OP (post author) and those with maintainer/admin privileges may mark as answer.

I am unable to mark answers in discussions for a repo I own.

Lmk what additional information would be helpful.

**Comment by DNin01** [comment]
Have you checked that answering is enabled in the category the post was in?

In such categories, you should see this button:
![Screenshot of a discussion comment. A button, labeled "Mark as answer", is outlined in dark orange.](https://docs.github.com/assets/cb-84381/mw-1440/images/help/discussions/comment-mark-as-answer-button.webp)

**Reply by olets** [reply]
How do I check that?

**Reply by DNin01** [reply, marked as answer]
> How do I check that?

You can see by clicking "New discussion" which categories are available and which ones have answers enabled, meaning that people can mark comments as answers. You can also set whether answering is enabled in your discussion categories by clicking the pencil icon over the category selection on the left, if you have the ability to.

**Reply by olets** [reply]
Thanks that was it
